source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.yml

```diff
@@ -274,13 +276,13 @@ tests:
             name: updateOne
             object: *collection
             arguments:
-              filter: 
-                $where": sleep(2000) || true
-              update: 
-                "$set": { "a": "bar" } 
+              filter:
+                $where: sleep(2000) || true
+              update:
+                "$set": { "a": "bar" }
```

Comment:
For future reference, none of these fields need to be quoted. If anything, quoting the value for `$where` a few lines up would make more sense since it contains some special characters (although still not required).